### ISSUE TITLE
Consistency across Filecoin + FIL references in documentation

### ISSUE THREAD

**Issue by har00ga** [filecoin-project/filecoin-docs#215]
It would be helpful to have consistency around the way that Filecoin (as in the network) and Filecoin (as in the token) are referred to within documentation. 'Filecoin' is used for both concepts interchangeably, which makes some of the concepts explained in the literature less clear than they could be. It would be much more optimal to refer to the token only as FIL and the network itself as Filecoin; this is the case across some of the existing pages, but not all of them. A good sweep of the existing content (especially the FAQ) and acknowledgement of this in the writing guide would be the best route to solving this.

**Comment by mishmosh** [comment]
Makes sense to me.

**Comment by pooja** [comment]
Makes sense to me as well -- also we have historically aspired to use `filecoin` for the token representation. `FIL` works too, but it's another way to differentiate in case people want to write out the full form. Agreed that including in the writing guide is a good step!

**Comment by johnnymatthews** [comment]
I suggest we use `FIL` since it mixes in well with things like `ETH`, `BTC`, etc. Did I see somewhere that there's a unicode symbol for `FIL`?

**Comment by johnnymatthews** [comment]
Found it: `⨎`

**Comment by Blazing-Mike** [comment]
I think this is reasonable. The network or protocol should be differentiated from the token. When we say Filecoin;
Filecoin is a Blockchain - Filecoin
Filecoin is a network or protocol - Filecoin
Filecoin is the token/crypto - FIL

**Comment by johnnymatthews** [comment]
Fixed in #1257.
